src/datachain/catalog/catalog.py

```diff
@@ -554,7 +554,9 @@ def __init__(
         self.metastore = metastore
         self._warehouse = warehouse
         self.cache = DataChainCache(datachain_dir.cache, datachain_dir.tmp)
-        self.client_config = client_config if client_config is not None else {}
+        self.client_config = (
+            client_config if client_config is not None else {"anon": True}
```

Comment:
hmm, either I'm missing something or would it set `anon` to True even if run it locally and credentials are provided via some env or profile config?

Reply:
I think that you're right, and that's precisely the point of this pull request. 🤔 Whether it's a good idea or not UX-wise, is a different topic. 😅 

Maybe we can set anonymous-by-default in Studio instead of here, just to fix the image loading bug?